packages/block-library/src/file/edit.js

```diff
@@ -271,16 +271,18 @@ function FileEdit( { attributes, isSelected, setAttributes, clientId } ) {
 					</ResizableBox>
 				) }
 				<div className={ 'wp-block-file__content-wrapper' }>
-					<RichText
-						tagName="a"
-						value={ fileName }
-						placeholder={ __( 'Write file name…' ) }
-						withoutInteractiveFormatting
-						onChange={ ( text ) =>
-							setAttributes( { fileName: text } )
-						}
-						href={ textLinkHref }
-					/>
+					<div className="wp-block-file__button-richtext-wrapper">
```

Comment:
Thanks for the PR @n2erjo00! We don't need to add an extra wrapper here. I don't remember the reason, but there where some issues with RichText and `inline` elements(a), so applying a `display:inline-block` would suffice.

Since we would need to target both front and back end, and there are different selectors, maybe something like that in `style.scss` could do the job.

```
.wp-block-file {
	a:not(.wp-block-file__button) {
		display: inline-block;
	}
}
```

Reply:
Cool 👍 You were right about this one. Added the css fix for this